ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -779,11 +779,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CFLAGS = "$(inherited)";
 				OTHER_CPLUSPLUSFLAGS = "$(inherited)";
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-Wl",
-					"-ld_classic",
-				);
+				OTHER_LDFLAGS = "$(inherited)";
```

Comment:
What Xcode version and CocoaPods version are you using?

Reply:
15.3 and 1.15.2